### PR TITLE
gui-libs/display-manager-init: revbump packages with init files referencing xdm

### PR DESCRIPTION
**Pull request by epsilon-0**
for stable packages added a new ebuild with revision bump, for unstable packages moved latest unstable with revision bump.

### CODE REVIEW

File: app-misc/g15stats/g15stats-1.9.7-r2.ebuild [hunk 1]
```diff
@@ -12,7 +12,6 @@ SRC_URI="mirror://sourceforge/g15daemon/${P}.tar.bz2"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
```
Comment (a17r) [inline app-misc/g15stats/g15stats-1.9.7-r2.ebuild:15]: packages with maintainers should not get unrelated changes made to them, even trivial ones, on such bulk changes.

**Review by Polynomial-C — APPROVED** [review]
